osu.Game/Overlays/Direct/PanelDownloadButton.cs

```diff
@@ -52,6 +51,8 @@ private void load(OsuGame game, BeatmapManager beatmaps)
                 return;
             }
 
+            noVideoSetting.BindTo(osuConfig.GetBindable<bool>(OsuSetting.PreferNoVideo));
```

Comment:
Yeah, this looks unsafe. `GetBindable` already returns a copy, and as far as I know bindables use weak references, so the inner bindable could go out of scope and get collected, which means this local copy won't get value updates.

To resolve, make `noVideoSetting` non-readonly and just assign:
```suggestion
            noVideoSetting = osuConfig.GetBindable<bool>(OsuSetting.PreferNoVideo);
```
(I wouldn't initialise `noVideoSetting` either, let it be `null` until `load()` is done)